### ISSUE TITLE
getOrFetch() doesn't use data parameter

### ISSUE THREAD

**Issue by 17h13** [AmpersandJS/ampersand-collection-rest-mixin#15]
When you call the fetch function you can pass a options.data parameter which will be used in the xhr call. This is working fine. But when you call the getOrFetch function with the options.all sets to true, eventually it will call the fetch function but in this call the options.data is ignored.

To correct the problem, I have replaced this code :
```
        if (options.all) {
            this.fetch({
                success: done,
                error: done
            });
        } else {

```
with this one 
```
        if (options.all) {
            options.success = done;
            options.error = done;
            this.fetch(options);
        } else {
```
I'm not sure it is perfect but If you want I can provide a pull request ... when I figure out how to write a test case.
Just tell me.

**Comment by kamilogorek** [comment]
I think it's a valid use case and would be nice to get it fixed.

**Comment by lukekarrys** [comment]
Closed by #15 and #19
